package.json

```diff
@@ -1137,12 +1137,12 @@
                 "docker.explorerRefreshInterval": {
                     "type": "number",
                     "default": 2000,
-                    "description": "Explorer refresh interval, default is 2000ms"
+                    "description": "Docker view refresh interval (milliseconds)"
                 },
                 "docker.containers.groupBy": {
                     "type": "string",
                     "default": "None",
-                    "description": "The property to use when grouping containers.",
+                    "description": "The property to use to group containers in Docker view: ContainerId, ContainerName, CreatedTime, FullTag, ImageId, Networks, Ports, Registry, Repository, RepositoryName, RepositoryNameAndTag, State, Status, Tag, or None",
```

Comment:
None is not specified in the enum value below. Is this an implicit value? Line 1290 defines a enum with None value.

Reply:
Never mind. It is defined, but I was looking at the bottom of the list.